contracts/staking/PoSValidatorManager.sol

```diff
@@ -355,19 +381,36 @@ abstract contract PoSValidatorManager is
         // initialize the removal.
         delegator.status = DelegatorStatus.PendingRemoved;
 
+        uint64 validatorUptimeSeconds;
         if (validator.status == ValidatorStatus.Active) {
+            if (includeUptimeProof) {
```

Comment:
Nit: Can we add a @notice comment that `includeUptimeProof` can always be `false` if the validation that the `delegationID` delegated to has already ended since the uptime proof will have already been provided?

Reply:
Added to the doc in `IPoSValidatorManager.sol`